styleguide-app/Examples/Modal.elm

```diff
@@ -52,50 +53,60 @@ example parentMessage state =
         , Modal.info
             { title = { title = "Modal.info", visibleTitle = state.visibleTitle }
             , wrapMsg = InfoModalMsg
-            , content =
-                viewContent
-                    state
-                    InfoModalMsg
-                    (Modal.primaryButton ForceClose "Continue")
+            , content = viewContent state InfoModalMsg Modal.primaryButton
             }
             state.infoModal
         , Modal.warning
             { title = { title = "Modal.warning", visibleTitle = state.visibleTitle }
             , wrapMsg = WarningModalMsg
-            , content =
-                viewContent
-                    state
-                    WarningModalMsg
-                    (Modal.dangerButton ForceClose "Have no fear")
+            , content = viewContent state WarningModalMsg Modal.dangerButton
             }
             state.warningModal
         ]
             |> List.map (Html.map parentMessage)
     }
 
 
-viewContent state wrapMsg primaryButton { firstFocusableElement, lastFocusableElement } =
+viewContent :
+    State
+    -> (Modal.Msg -> Msg)
+    -> (Msg -> String -> List (Root.Attribute Msg) -> Html Msg)
```

Comment:
🎨  could Modal define a union type of `Modal.Style = Danger | Primary | Secondary`  or even reuse Button.V8.ButtonStyle to avoid the need to pass a function as a configuration argument?